src/walletSdk/Horizon/Transaction/TransactionBuilder.ts

```diff
@@ -90,6 +95,91 @@ export class TransactionBuilder extends CommonTransactionBuilder<TransactionBuil
     return this;
   }
 
+  /**
+   * Creates and adds a path payment operation to the transaction builder.
+   *
+   * @param {string} destinationAddress - The destination Stellar address to which the payment is sent.
+   * @param {StellarAssetId} sendAsset - The asset to be sent.
+   * @param {StellarAssetId} destAsset - The asset the destination will receive.
+   * @param {string} [sendAmount] - The amount to be sent. Must specify either sendAmount or destAmount,
+   * but not both.
+   * @param {string} [destAmount] - The amount to be received by the destination. Must specify either sendAmount or destAmount,
+   * but not both.
+   * @param {string} [destMin] - The minimum amount of the destination asset to be receive. This is a
+   * protective measure, it allows you to specify a lower bound for an acceptable conversion. Only used
+   * if using sendAmount.
+   * (optional, default is ".0000001").
+   * @param {string} [sendMax] - The maximum amount of the destination asset to be sent. This is a
+   * protective measure, it allows you to specify an upper bound for an acceptable conversion. Only used
+   * if using destAmount.
+   * (optional, default is int64 max).
+   *
+   * @returns {TransactionBuilder} - Returns the current TransactionBuilder instance for method chaining.
+   */
+  pathPay({
+    destinationAddress,
+    sendAsset,
+    destAsset,
+    sendAmount,
+    destAmount,
+    destMin,
+    sendMax,
+  }: PathPayParams): TransactionBuilder {
+    if ((sendAmount && destAmount) || !(sendAmount || destAmount)) {
```

Comment:
[nit] for me (!send && !dest) is a bit more readable

Reply:
👍  https://github.com/stellar/typescript-wallet-sdk/pull/56/commits/3f35881da6bb54cbaa5dcc9a9d2994f5dd2d7aed